src/main/java/seedu/knowitall/logic/commands/SearchCommand.java

```diff
@@ -28,7 +30,12 @@ public SearchCommand(QuestionContainsKeywordsPredicate predicate) {
     }
 
     @Override
-    public CommandResult execute(Model model, CommandHistory history) {
+    public CommandResult execute(Model model, CommandHistory history) throws CommandException {
+
+        if (model.getState() != State.IN_FOLDER && model.getState() != State.IN_HOMEDIR) {
```

Comment:
The search command does not work outside of a folder!
```suggestion
        if (model.getState() != State.IN_FOLDER) {
```